tests/test_store.py

```diff
@@ -218,6 +218,32 @@ def test_getDatasetManifest(self, synapse_store, downloadFile):
             # return manifest id
             assert manifest_data == "syn51204513"
 
+    def test_synapse_rest_request(self, synapse_store):
```

Comment:
@GiaJordan since I have merged my PR into your PR, this test will need to be updated?

Reply:
We'll remove it along with `send_api_request`

Reply:
that change has been pushed now